css/src/components/admin-list.css

```diff
@@ -1,45 +1,76 @@
 /**
- * Admin lists.
+ * @file
+ * Admin list and admin item modules.
  */
-ul.admin-list {
-  margin: 0;
+
+/**
+ * Admin list (wrapper of admin items).
+ */
+.admin-list {
+  margin-top: 1em;
+  margin-bottom: 2em;
   padding: 0;
+  border: calc(1em / 16) solid var(--color-lightgray);
+  border-right: 0;
+  border-left: 0;
+}
+.admin-list--panel {
+  margin: -1em -1.5em;
+  border-top: 0;
+  border-bottom: 0;
+}
+
+/**
+ * Admin item.
+ */
+.admin-item {
+  padding: 0.75em 1.5em;
+}
+.admin-item--expanded {
+  padding-left: 3em;
 }
-.admin-list li {
+[dir="rtl"] .admin-item--expanded {
+  padding-right: 3em;
+  padding-left: 1.5em;
+}
+.admin-item + .admin-item {
+  border-top: calc(1em / 16) solid var(--color-lightgray);
```

Comment:
Why isn't this just `.admin-item`? If we change this to border-bottom, this would match the design in the examples.

@ckrina I think there's a mistake in the component spec stating that there should be `border-top` but it doesn't exist in any of the examples.

Reply:
@lauriii agree. We've removed the border top, but also the border bottom in lists without header. We've also updated the specs and examples.

Reply:
We have two kind of admin list: first is a _raw_ one, for example pls visit `/admin`. The second type is when the admin list is the content of a panel like on `/admin/index` or at `/admin/config/`.

When the admin list is rendered inside a panel, the bottom border is the part of the őanle component.

Reply:
I would prefer to work this the other way around to match with the styleguide. `.admin-item` has always `border-bottom`, but it is removed from `.admin-item:last-child` if not placed inside `.admin-list--panel`. Not nice as well but there's no clean way to theme this kind of complex designs 🤷‍♂️

Reply:
We have two components here: the panel and the admin-list.

I think that the bottom border should be provided by the panel component and not by the admin list.

I see that you insist to the `:last-child` approach, but I dont want to remove the bottom border from the panel since that's the only visual indication where the panel content "ends".

Reply:
@lauriii this is updated in drupalux/claro@331a13b0c0, pls review